sprokit/processes/adapters/input_adapter_process.cxx

```diff
@@ -77,16 +77,16 @@ kwiver::adapter::ports_info_t
 input_adapter_process
 ::get_ports()
 {
-  kwiver::adapter::ports_info_t port_info;
+  kwiver::adapter::ports_info_t l_port_info;
```

Comment:
I'd make this consistent with the change in `output_adapter_process.cxx` and others. (I'd also probably use "π"... ":pie:"... er, "pi" :slightly_smiling_face:.)

(On that note, I do believe C++23 is going to forbid use of emoji as identifier names, so don't use ":pie:" :wink:.)